openssl/src/pkcs7.rs

```diff
@@ -281,10 +281,31 @@ impl Pkcs7Ref {
 
         Ok(())
     }
+
+    /// Retrieve the signer's certificates from the PKCS#7 structure without verifying them.
+    ///
+    /// This corresponds to [`PKCS7_get0_signers`].
+    ///
+    /// [`PKCS7_get0_signers`]: https://www.openssl.org/docs/man1.0.2/crypto/PKCS7_verify.html
+    pub fn signers(
+        &self,
+        certs: &StackRef<X509>,
+        flags: Pkcs7Flags,
+    ) -> Result<&StackRef<X509>, ErrorStack> {
```

Comment:
The returned stack is created in PKCS7_get0_signers and needs to be freed by the caller, so this should return a `Result<Stack<X509>, ErrorStack>`.